packages/tinacms/src/cache/node-cache.ts

```diff
@@ -6,12 +6,31 @@ const makeKey = (key: any) => {
   return createHash('sha256').update(input).digest('hex')
 }
 
-export const NodeCache = (dir: string, fs: any): Cache => {
+// makeCacheDir creates the cache directory if it doesn't exist
+const makeCacheDir = (dir: string, fs: any) => {
+  const path = require('path')
+  const os = require('os')
```

Comment:
Why do we need to use `require` here?

Reply:
This is conditionally imported depending on whether it is a nodejs env.

The only alternative (without polyfilling) is dynamic imports - I wasn't able to get them working correctly though